hyper/cmd/pack.go

```diff
@@ -86,7 +86,7 @@ var stopCmd = &cobra.Command{
 	},
 }
 var hyperpackageCmd = &cobra.Command{
```

Comment:
One other thing I thought of-- we might want to change `hyperpackageCmd` to just `packCmd`